### PR TITLE
Update to WebRTC 5993 (M118)

### PR DESCRIPTION
**Pull request by inaqui-signal**

### CODE REVIEW

File: sdk/android/native_api/audio_device_module/audio_device_android.cc [hunk 1]
```diff
@@ -125,7 +125,7 @@ rtc::scoped_refptr<AudioDeviceModule> CreateJavaAudioDeviceModule(
       AudioDeviceModule::kAndroidJavaAudio, false /* use_stereo_input */,
       false /* use_stereo_output */,
       jni::kHighLatencyModeDelayEstimateInMilliseconds, std::move(audio_input),
-      std::move(audio_output)
+      std::move(audio_output),
```
Comment (inaqui-signal) [inline sdk/android/native_api/audio_device_module/audio_device_android.cc:128]: TODO: This syntax error has been here for a while, even in main, but it looks like this is now being compiled which led to a compile time error. Need to investigate why and if this should be compiled for the Android builds.

File: p2p/client/basic_port_allocator.cc [hunk 2]
```diff
@@ -281,8 +281,10 @@ BasicPortAllocator::CreateIceGatherer(const std::string& name) {
   // for IceGatherers.
 
   // 1. Create with NetworkManager, PacketSocketFactory, and RelayPortFactory.
-  auto new_allocator = std::make_unique<BasicPortAllocator>(network_manager(), socket_factory_.get());
-  new_allocator->Init(relay_port_factory_);
+  auto new_allocator = std::make_unique<BasicPortAllocator>(network_manager(),
```
Comment (inaqui-signal) [inline p2p/client/basic_port_allocator.cc:284]: TODO: confirm this is the correct way of initializing the BasicPortAllocator

File: audio/channel_receive.cc [hunk 3]
```diff
@@ -70,14 +70,16 @@ acm2::AcmReceiver::Config AcmConfig(
     rtc::scoped_refptr<AudioDecoderFactory> decoder_factory,
     absl::optional<AudioCodecPairId> codec_pair_id,
     size_t jitter_buffer_max_packets,
-    bool jitter_buffer_fast_playout) {
+    bool jitter_buffer_fast_playout,
+    int jitter_buffer_min_delay_ms) {
```
Comment (inaqui-signal) [inline audio/channel_receive.cc:74]: Not sure if this is of interest as it relates to that jitter buffer issue on mobile.

Comment (rashad-signal) [reply audio/channel_receive.cc:74]: This looks like it's for forcing a minimum delay, but it defaults to 0, so we should be fine.

**Review by jim-signal — APPROVED** [review]

**Review by rashad-signal — APPROVED** [review]

**Pull request merged**